packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -644,19 +644,19 @@ export class DeltaManager
                         logNetworkFailure(
                             this.logger,
                             {
-                                delay, // seconds
+                                delayMs, // milliseconds
```

Comment:
```suggestion
                               delay: delayMs, // milliseconds
```
let's not change the property we log, can break queries